src/components/slideshow/slideshow.js

```diff
@@ -188,6 +188,11 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
                 stopInterval();
             });
 
+            // Blur foreign element to prevent starting of "nested" slideshow
+            if (document.activeElement && !dlg.contains(document.activeElement)) {
+                document.activeElement.blur();
+            }
```

Comment:
This can happen with any dialog. Maybe the best place to add this change would be the dialog component instead.